pages/index.js

```diff
@@ -19,27 +20,45 @@ export async function getServerSideProps() {
   const URL =
```

Comment:
Thought: if you do SSR for these many users(in future), the initial load time will be too much. I think hybrid rendering is better. If SEO is a concern, render a loading view from server and fetch the data in client. 
@ankushdharkar @invalidtoken